python/lib/sift_py/rule/config.py

```diff
@@ -245,6 +187,26 @@ def from_str(cls, val: str) -> Optional["RuleActionKind"]:
         return None
 
 
+class RuleActionAnnotationKind(Enum):
```

Comment:
FYI on this one -- I originally had moved this due to it creating a circular dependency, then after re-arranging things on this PR it was circular again... so I moved it back, and things seem to work fine now.